app/models/user.rb

```diff
@@ -14,18 +14,17 @@ class User < ActiveRecord::Base
   # Makes getters and setters
   attr_accessor :pin
 
-  validates :school, :first_name, :last_name, :presence => true
-  # Validation only occurs when a user record is being
+  # Validation's for email and pin only occurs when a user record is being
   # created on sign up. Does not occur when updating
   # the record.
+  validates :school_id, :first_name, :last_name, :presence => true
   validates_format_of :email, with: /\@schools.nyc\.gov/, message: ' should end in @schools.nyc.gov', :on => :create
-  validates_format_of :first_name, :last_name, :with => /^[a-z]+$/i
+  validates_format_of :first_name, :last_name, :with => /\A[^0-9`!@;#\$%\^&*+_=\x00-\x19]+\z/
   validates_format_of :alt_email,:with => Devise::email_regexp, :allow_blank => true, :allow_nil => true
-  # Validation only occurs when a user record is being
-  # created on sign up. Does not occur when updating
-  # the record.
+  validates :alt_email, uniqueness: true, allow_blank: true, allow_nil: true
```

Comment:
Is the doc string deleted because it;s no longer true?  If so, could we modify the doc string, rather than delete?

Reply:
The doc string was being presented twice. I edited line 17 to be specific. It's cleaner this way.